src/components/AssigneeSelector.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <select v-model="assignee" @change="onChange">
+      <option value="null">No Assignee</option>
+      <option
+        v-for="user in users"
+        :key="user.gid"
+        :value="user.gid"
+      >{{ user.name }}</option>
+    </select>
+  </div>
+</template>
+
+<script lang="ts">
+import store from "@/store";
+import { defineComponent, onMounted, ref } from "vue";
+import { createNamespacedHelpers } from "vuex";
+const { mapState } = createNamespacedHelpers("asana");
+
+export default defineComponent({
+  computed: {
+    ...mapState(["users"]),
+  },
+  setup() {
+    const assignee = ref<string | null>();
+
+    onMounted(() => {
+      assignee.value = store.state["preferences"].taskEditorSectionIdAndTask.task.assignee?.gid ?? null;
+    })
+
+    return {
+      assignee
+    };
+  },
+  methods: {
+    onChange(event: Event) {
```

Comment:
Vue3 approach is to make this a function that gets returned from setup. methods{} is a vue2 thing